src/main/java/com/exactpro/epfast/template/implementations/AsciiStringFieldClass.java

```diff
@@ -0,0 +1,17 @@
+package com.exactpro.epfast.template.implementations;
+
+import com.exactpro.epfast.template.AsciiStringField;
+
+public class AsciiStringFieldClass extends FieldInstructionClass implements AsciiStringField {
```

Comment:
Remove Class from all names. It should be okay as we only use interfaces with same names in `implements` part where they become fully qualified names which is acceptable